### DISCUSSION TITLE
Custom sketchfab tabs in media browser

### DISCUSSION THREAD

**Discussion by safeisnotanoption** [mozilla/hubs#4223] [General Questions & Discussion] [answered]
Is it possible to embed my own sketchfab collections so that they appear in the media browser in their own tabs?

<a href="https://ibb.co/QPy9K4m"><img src="https://i.ibb.co/48hWVHN/customtab.jpg" alt="customtab" border="0"></a>

**Comment by j-conrad** [comment, marked as answer]
This should be theoretically possible with your own Hubs Cloud. I'm not an expert on the Sketchfab API, but displaying models from a specific account should be doable.
Allowing individual users to access _their_ accounts might be trickier and I'm not sure what Sketchfab's terms of service say about that sort of thing so you may want to ask on their forums.
